HTML,CSS, and Java-Script additions

DaneLarsen1 opened the pull request:
Created a custom spotify holiday logo for us to use.

Cleaned up the font sizing for a cleaner look.

Added functionality to the "select a holiday" dropdown box so that it'll show the corresponding page to the right which will say the holiday name and will give us the correct format to add previews to songs and lyrics through Spotify

Made the title hug the logo with a margin of 20px

DaneLarsen1 committed:
Pushing entire base html and css for our webpage

Day 1 full class project work

DaneLarsen1 committed:
HTML,CSS, and Java-Script additions

Created a custom spotify holiday logo for us to use.
Cleaned up the font sizing for a cleaner look.
Added functionality to the "select a holiday" dropdown box so that itll show the corrisponding page to the right which will say the holiday name and will give us the correct format to add previews to songs and lyrics through spotify
Made the title hug the logo with a margin of 20px

HoneyBuzz94 committed:
Added link to jQuery

HoneyBuzz94 committed:
Added links to DayJS and Bootstrap

HoneyBuzz94 committed:
Added rounding to the boxes in main. Edited the text and CSS for the footer.

HoneyBuzz94 committed:
Added CSS to make the page responsive. Main flex direction changes to column when 768px

ruthiepina commented on Images/spotify-high-resolution-logo-transparent.png:
the logo is amazing!!

TNielson78 reviewed (approved):
Looks great logo is awesome....

DaneLarsen1 committed:
Front side and javascript updates

Changes Made

1. Changed the color for each holiday individually when clicked
2. Changed the view and colors to match spotify when sites first loaded
3. Changed some small layout issues to make the code easier for the coders.

DaneLarsen1 committed:
fixed a console error on webpage

Deleted one unused line
